src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -705,7 +705,8 @@ protected override BoundNode VisitReturnStatementNoAdjust(BoundReturnStatement n
                 return null;
             }
 
-            expr = RemoveImplicitConversions(expr);
+            Conversion conversion;
```

Comment:
Not related to this PR: to my knowledge, this is the first occurrence of a mixed deconstruction (one existing variable and one freshly declared one) in Roslyn ;-)